doc/source/programs/gdal_raster_footprint.rst

```diff
@@ -0,0 +1,175 @@
+
+.. option:: --srcnodata <value>
+
+    Set nodata values for input bands (different values can be supplied for each band).
+    If a single value is specified, it applies to all selected bands.
+    If more than one value is supplied, there should be as many values as the number
+    of selected bands, and all values should be quoted to keep them
+    together as a single operating system argument.
+    If the option is not specified, the intrinsic mask band of each selected
+    bands will be used.
+
+.. option:: --coordinate-system pixel|georeferenced
+
+    Target coordinate system. By default if the input dataset is georeferenced,
+    ``georeferenced`` is implied, that is the footprint geometry will be expressed
+    as coordinates in the CRS of the raster (or the one specified with :option:`--dst-crs`).
+    If specifying ``pixel``, the coordinates of the footprint geometry are
+    column and line indices.
+
+.. option:: --dst-crs <CRS_DEF>
+
+    Target CRS of the output file.  The <CRS_DEF> may be any of
+    the usual GDAL/OGR forms, complete WKT, PROJ.4, EPSG:n or a file containing
+    the WKT.
+    Specifying this option implies ``--coordinate-system=georeferenced``.
+    The footprint is reprojected from the CRS of the source raster to the
+    specified CRS.
+
+.. option:: --split-multipolygons
+
+    When specified, multipolygons are split as several features each with one
+    single polygon.
+
+.. option:: --convex-hull
+
+    When specified, the convex hull of (multi)polygons is computed.
+
+.. option:: --densify-distance <value>
+
+    The specified value of this option is the maximum distance between 2
+    consecutive points of the output geometry.
+    The unit of the distance is in pixels if :option:`--coordinate-system` equals ``pixel``,
+    or otherwise in georeferenced units of the source raster.
+    This option is applied before the reprojection implied by :option:`--dst-crs`.
+
+.. option:: --simplify-tolerance <value>
+
+    The specified value of this option is the tolerance used to merge
+    consecutive points of the output geometry using the
+    :cpp:func:`OGRGeometry::Simplify` method.
+    The unit of the distance is in pixels if :option:`--coordinate-system` equals ``pixel``,
+    or otherwise in georeferenced units of the target vector dataset.
+    This option is applied after the reprojection implied by :option:`--dst-crs`.
+
+.. option:: --min-ring-area <value>
+
+    Minimum value for the area of a ring
+    The unit of the area is in square pixels if :option:`--coordinate-system` equals ``pixel``,
+    or otherwise in georeferenced units of the target vector dataset.
+    This option is applied after the reprojection implied by :option:`--dst-crs`
+
+.. option:: --max-points <value>|unlimited
+
+    Maximum number of points of each output geometry (not counting the closing
+    point of each ring, which is always identical to the first point).
```

Comment:
On line 112 at least 4 points are required. I thought it means 3 points for a triangle + one more for closing.

Reply:
Given that rasterization at least generate one square, you need at least 4 points